scripts/tools-protobuf.sh

```diff
@@ -0,0 +1,7 @@
+curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v3.12.0/protoc-3.12.0-linux-x86_64.zip
+unzip protoc-3.12.0-linux-x86_64.zip -d $HOME/.local
+export PATH="$PATH:$HOME/.local/bin"
```

Comment:
Do not add extra things on PATH, add it to /usr/local/bin or something like that